package/gluon-client-bridge/luasrc/lib/gluon/upgrade/320-gluon-client-bridge-wireless

```diff
@@ -84,7 +84,7 @@ local function configure_owe_transition_mode(config, radio_name)
 		return
 	end
 
-	if not ap() or not ap.owe_transition_mode() then
+	if not ap() or (ap.owe_transition_mode() ~= nil and not ap.owe_transition_mode()) then
```

Comment:
Both nil as well as false result in a boolean false in this expression. So why you feel this is necessary?

Reply:
I wanted to change the default to true here. How would be the correct syntax?

Reply:
and --> or

Reply:
`not ap() or not ap.owe_transition_mode()` is already unnecessarily verbose. For default `false` use:
```lua
if not ap.owe_transition_mode(false) then
```
To make the default true, just change that to:
```lua
if not ap.owe_transition_mode(true) then
```
Our `site` library allows arbitrary indexing into non-existing site.conf keys - `ap.this.key.is.nonsense('foo')` would just return `'foo'` without throwing an error.

Unrelated to this PR, the same simplification is possible few other places in this file.

Reply:
OK, I will do an extra PR for that. and I reverted the default change

Reply:
#2031